workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml

```
@@ -10,13 +10,19 @@ building_count:
   endpoint: elements/count
   filter: building=* and geometry:polygon
   ratio_filter: building=* and geometry:polygon and (height=* and height!="") or (building:levels=* and building:levels!="")
+  indicators:
+    - MappingSaturation
+    - Currentness
+    - AttributeCompleteness
```

Please use lower hyphen case (related to #386)
```suggestion
    - mapping-saturation
    - currentness
    - attribute-completeness
```

---

The same for all other lists of indicator keys